tendenci/apps/memberships/utils.py

```diff
@@ -1211,15 +1217,12 @@ def clean_membership_type(self, memb_data, **kwargs):
                 else:
                     memb_data['membership_type'] = value
             else:
-                if not MembershipType.objects.filter(
-                                            name=value).exists():
+                candidates = MembershipType.objects.filter(name=value)
+                if not candidates.exists():
```

Comment:
Simple DRY improvement. 
We keep a copy of `MembershipType.objects.filter(name=value)` in `candidates` for reuse in the `else` block rather than recoding the query.